src/dom/dom.js

```diff
@@ -1463,15 +1463,28 @@ p5.prototype.createSelect = function(...args) {
  * @example
  * <div>
  * <code>
+ * let style = document.createElement('style');
+ * style.innerHTML = `
+ * .p5-radio label {
+ *    display: flex;
+ *    align-items: center;
+ *  }
+ *  .p5-radio input {
+ *    margin-right: 5px;
+ *  }
+ *  `;
+ * document.head.appendChild(style);
+ *
  * let myRadio;
  *
  * function setup() {
  *   createCanvas(100, 100);
  *
  *   // Create a radio button element and place it
- *   // in the top-left corner.
+ *   // in the top-right corner.
```

Comment:
Okay, so we are rendering in a 2D canvas and it's coordinates at 0,0 represents a top-left corner. If it was a webgl canvas (0,0) represented the mid of the canvas. So it should be top-left